src/app/routes/create-account/create-account.component.ts

```diff
@@ -25,17 +25,29 @@ export class CreateAccountComponent implements OnInit {
   emailPhoneType: any
   otpPage = false
   // errors: any
-  spherFormBuilder: FormBuilder
-  public createAccountForm: FormGroup
-  public otpCodeForm: FormGroup
+  // spherFormBuilder: FormBuilder
+  createAccountForm: FormGroup
+  otpCodeForm: FormGroup
 
   constructor(
-    spherFormBuilder: FormBuilder,
+    private spherFormBuilder: FormBuilder,
     private snackBar: MatSnackBar,
     private signupService: SignupService,
     private router: Router
   ) {
-    this.spherFormBuilder = spherFormBuilder
+    // this.spherFormBuilder = spherFormBuilder
```

Comment:
this form creation we need in method @AnkitaTarento  please confirm it . to fix this we need to just change in tsconfig.json